dataflux_pytorch/dataflux_mapstyle_dataset.py

```diff
@@ -200,3 +208,20 @@ def _list_GCS_blobs_with_retry(self):
         # raised an exception.
         else:
             raise error
+
+    def _has_permissions(self):
```

Comment:
Is there any way to share this implementation between the iterable and map-style data sets instead of duplicating them?

Reply:
Done.